sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
 # Release History
 
-## 1.0.0-preview.1 (Unreleased)
\ No newline at end of file
+## 1.0.0-preview.1 (Unreleased)
```

Comment:
Can we fill in more here? Perhaps Sejal can help?

Reply:
I will ask her

Reply:
She's OOF today. I'd add what you think makes sense to start, and she can offer edits when she returns if she has any.

Maybe peruse a few other teams' changelogs for their first release to find prior art?

Reply:
Most teams have this initial release log empty. 
I added some template info and an initial log to not have it empty